datalad/cmdline/tests/test_main.py

```diff
@@ -90,13 +90,18 @@ def test_help_np():
     # none of the lines must be longer than 80 chars
     # TODO: decide on   create-sibling and possibly
     # rewrite-urls
+    import shutil
+    accepted_width = shutil.get_terminal_size()[0] \
+        if hasattr(shutil, 'get_terminal_size') else 80
```

Comment:
"not sure" what we are "fixing" with this. we shouldn't test for less than 80... and not sure if valuable to test for > 80